File: loop_transformations/sources/apply_syntax.2.c
/*
* @@name:   apply_syntax.2
* @@type:   C
* @@operation: compile
* @@expect:	success
* @@version:	omp_6.0
*/
void apply_assoc(double A[100][100][3])
{
   #pragma omp tile sizes(4,4) \
               apply(     grid: interchange,nothing) \
               apply(intratile: nothing,interchange)
   for (int i = 0; i < 100; ++i)
   for (int j = 0; j < 100; ++j)

      // k loop not associated with tile, but with interchange
      for (int k = 0; k < 3; ++k)
         A[i][j][k] = A[i][j][k] + 1;
}

File: loop_transformations/sources/apply_syntax.2.f90
! @@name:       apply_syntax.2
! @@type:       F-free
! @@operation:  compile
! @@expect:     success
! @@version:    omp_6.0
subroutine apply_assoc(A)
   implicit none
   double precision :: A(0:2, 0:99, 0:99)
   integer          ::    k,   j,    i   

   !$omp  tile sizes(4,4) &
   !$omp&      apply(     grid: interchange, nothing) &
   !$omp&      apply(intratile: nothing, interchange)
   do i = 0, 99
   do j = 0, 99

      do k = 0, 2 !! k loop not associated with tile, but w. interchange
         A(k,j,i) = A(k,j,i) + 1
      enddo
   enddo; enddo
end subroutine
